spec/features/comments/user_creates_new_comment_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe "user fills out comment form on job page" do
```

Comment:
This looks great. Just two notes:

* Based on the spec I would probably add a test to see that new comments render in the proper order. Take a look at `nth-of-type` and `nth-child` CSS selectors for that test.
* Consistent with the SO article Andrew posted, would recommend using `expect(page).to have_content("Cool job")` over `.has_content?`